Packs/CommonScriptsV2/Scripts/GetIndexOfArrayValue/GetIndexOfArrayValue.yml

```diff
@@ -0,0 +1,29 @@
+args:
+- description: The value is an array of elements
+  isArray: true
+  name: value
+- description: Provide the value from the array
+  name: array_value
+  required: true
+comment: |-
+  This transformer will get an index of an element from an array.
+  ex:["phishing","Malware"], if we provide "Malware" to array_value argument, will get index as 1.
```

Comment:
```suggestion
comment: "Retrieve the index of a given item in a given array.\nex:['phishing', 'Malware'], if we provide 'Malware' to array_value argument, will get index as 1.
```

Reply:
I haven't provided the tags:transformer and i have made this script to use as transformer